### ISSUE TITLE
Update to use Django 1.7 / 1.8

### ISSUE THREAD

**Issue by aaaaalbert** [SeattleTestbed/custominstallerbuilder#6]
The custominstallerbuilder app currently runs on Django 1.6 (which just had its final, end-of-life version released). Django 1.8 will be the new Long-Term Supported version, so this is should be our goal.

(See SeattleTestbed/clearinghouse#152 for potential constraints, and first tests.)

**Comment by lukpueh** [comment]
+1

Here are some deprecation warning:
```sh
[Wed May 18 14:07:05.876024 2016] [wsgi:error] [pid 6105:tid 139860982261504] sys:1: DeprecationWarning: Creating streaming responses with `HttpResponse` is deprecated. Use `StreamingHttpResponse` instead if you need the streaming behavior.
[Wed May 18 14:07:05.876168 2016] [wsgi:error] [pid 6105:tid 139860982261504] 
[Wed May 18 14:20:26.302737 2016] [wsgi:error] [pid 6105:tid 139860906727168] /home/cib/.virtualenvs/cib/lib/python2.7/site-packages/django/http/response.py:330: DeprecationWarning: Using mimetype keyword argument is deprecated, use content_type instead
[Wed May 18 14:20:26.302903 2016] [wsgi:error] [pid 6105:tid 139860906727168]   super(HttpResponse, self).__init__(*args, **kwargs)
```

**Comment by lukpueh** [comment]
Some more reasons to update django I found in the log:
RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
RemovedInDjango110Warning: django.core.context_processors is deprecated in favor of django.template.context_processors.

**Comment by lukpueh** [comment]
When fixing the problem in https://github.com/SeattleTestbed/custominstallerbuilder/pull/21 I could not replicate the last three Warnings from above log dump.

**Comment by aaaaalbert** [comment]
Here's the desired grep from alpha-ch (Python 2.7.6, Django 1.8.5):
```
RemovedInDjango110Warning: django.core.context_processors is deprecated in favor of django.template.context_processors.
 RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
 RemovedInDjango110Warning: You haven't defined a TEMPLATES setting. You must do so before upgrading to Django 1.10. Otherwise Django will be unable to load templates.
```

That said, #21 looks good and yields a working CIB.

**Comment by aaaaalbert** [comment]
1.8 compatibility fixed in #21. 1.9/1.10 compatibility is a separate issue.
